### DISCUSSION TITLE
在该程序的 vid2vid.py 文件的第 116 行中，尝试访问processed_frames 列表的第一个元素（即索引值为0），但是该列表没有任何元素，因此会引发 IndexError。

### DISCUSSION THREAD

**Discussion by tozige** [volotat/SD-CN-Animation#81] [General]
An exception occurred while trying to process the frame: list index out of range
Traceback (most recent call last):
  File "D:\sd-webui-aki-v4\extensions\SD-CN-Animation\scripts\base_ui.py", line 114, in process
    yield from vid2vid.start_process(*args)
  File "D:\sd-webui-aki-v4/extensions/sd-cn-animation/scripts\core\vid2vid.py", line 116, in start_process
    processed_frame = np.array(processed_frames[0])
IndexError: list index out of range

**Comment by volotat** [comment]
There have already been several discussions about the issue. Please check them out, as you may find something that will help you. https://github.com/volotat/SD-CN-Animation/issues/64 https://github.com/volotat/SD-CN-Animation/issues/73 https://github.com/volotat/SD-CN-Animation/issues/32
